proto/services/tasks_svc/v1/properties_svc.proto

```diff
@@ -0,0 +1,271 @@
+  optional string set_id = 8;
+
+  message SelectOption {
+    string name = 1;
+    optional string description = 2;
+  }
+  repeated SelectOption select_options = 9;
+}
+
+//
+// GetProperties
+//
+
+message GetPropertiesRequest {
+  optional string set_id = 1; // @gotags: validate:"uuid4"
+
+  // TODO: TBD: other filters / search params, e.g. include_is_archived
+}
+
+message GetPropertiesResponse {
+  message Property {
+    string id = 1;
+    SubjectType subject_type = 2;
+    FieldType field_type = 3;
+
+    string name = 4;
+    optional string description = 5;
+
+    bool is_archived = 6;
+    bool is_soft_required = 7;
+
+    // TODO: TBD: priortyDisplayedAtSubjectIds?
+
+    // ID of set this Property should belong to
+    optional string set_id = 8;
+
+    message SelectOption {
+      string name = 1;
+      optional string description = 2;
+    }
+    repeated SelectOption select_options = 9;
+  }
+
+  repeated Property properties = 1;
+}
+
+//
+// UpdateProperty
+//
+
+message UpdatePropertyRequest {
+  string id = 1;
+
+  optional SubjectType subject_type = 2;
+  optional FieldType field_type = 3;
+
+  optional string name = 4; // TODO: validation
+  optional string description = 5; // TODO: validation
+
+  optional bool is_archived = 6;
+  optional bool is_soft_required = 7;
+
+  // TODO: TBD: priortyDisplayedAtSubjectIds?
+
+  // ID of set this Property should belong to
+  optional string set_id = 8;
+
+  message SelectOption {
+    string id = 1;
+    optional string name = 2;
+    optional string description = 3;
+  }
+  repeated SelectOption new_or_updated_select_options = 9;
+  repeated string remove_select_options = 10; // @gotags: validate:"dive,uuid4"
+}
+
+message UpdatePropertyResponse {}
+
+/***************************************************/
+/************ Work with Property Values ************/
+/***************************************************/
+
+//
+// AttachPropertyValue
+//
+
+// attach the same property again to update
+message AttachPropertyValueRequest {
+  string subject_id = 1; // @gotags: validate:"uuid4"
+  SubjectType subject_type = 2; // TODO: validation (not 0)
+  string property_id = 3; // @gotags: validate:"uuid4"
+
+  oneof value {
+    bool is_undefined = 4; // if it exists, it should be true
+    string text_value = 5; // FIELD_TYPE_TEXT
+    float number_value = 6; // FIELD_TYPE_NUMBER
+    bool bool_value = 7; // FIELD_TYPE_CHECKBOX
+    string date_value = 8; // FIELD_TYPE_DATE TODO: is there a proto date type?
+    string date_time_value = 9; // FIELD_TYPE_DATE_TIME
+    string select_value = 10; // FIELD_TYPE_SELECT, id of option
+  }
+}
+
+message AttachPropertyValueResponse {}
+
+//
+// GetAttachedPropertyValues
+//
+
+message GetAttachedPropertyValuesRequest {
+  string subject_id = 1; // @gotags: validate:"uuid4"
+
+  optional string view = 2;
+  // TODO: TBD: other filters / search params, e.g. include_is_archived
+}
+
+message GetAttachedPropertyValuesResponse {
```

Comment:
Based on https://github.com/helpwave/services/pull/617, this response will be quite hard to achieve without projections. The init PR will introduce `Property`, `PropertySet` and `PropertyValue` as separate gRPC-services.

`GetAttachedPropertyValues` can stay for now. Could you introduce `GetPropertyValue` that returns the scoped data of just the `PropertyValue` service?